src/network_analysis.jl

```diff
@@ -645,25 +709,43 @@ end
 
 # Used in the subsequent function.
 function cache_conservationlaw_eqs!(rn::ReactionSystem, N::AbstractMatrix, col_order)
+    # Retrieves nullity (the number of conservation laws). `r` is the rank of the netstoichmat.
     nullity = size(N, 1)
-    r = numspecies(rn) - nullity     # rank of the netstoichmat
-    sts = species(rn)
+    r = numspecies(rn) - nullity
+
+    # Creates vectors of all independent and dependent species (those that are not, and are, 
+    # eliminated by the conservation laws). Get vectors both with their indexes and the actual
+    # species symbolic variables.
+    sps = species(rn)
     indepidxs = col_order[begin:r]
-    indepspecs = sts[indepidxs]
+    indepspecs = sps[indepidxs]
     depidxs = col_order[(r + 1):end]
-    depspecs = sts[depidxs]
+    depspecs = sps[depidxs]
+
+    # Declares the conservation law parameters.
     constants = MT.unwrap.(MT.scalarize(only(
                 @parameters $(CONSERVED_CONSTANT_SYMBOL)[1:nullity] [conserved=true])))
 
+    # Computes the equations for (examples uses simple two-state system, `X1 <--> X2`):
+    # - The species eliminated through conservation laws (`conservedeqs`). E.g. `[X2 ~ X1 - Γ[1]]`.
+    # - The conserved quantity parameters (`constantdefs`). E.g. `[Γ[1] ~ X1 + X2]`.
     conservedeqs = Equation[]
     constantdefs = Equation[]
+
+    # For each conserved quantity.
     for (i, depidx) in enumerate(depidxs)
+        # Finds the coefficient (in the conservation law) of the species that is eliminated
+        # by this conservation law.
         scaleby = (N[i, depidx] != 1) ? N[i, depidx] : one(eltype(N))
         (scaleby != 0) || error("Error, found a zero in the conservation law matrix where "
-              *
-              "one was not expected.")
+            * "one was not expected.")
+
+        # Creates, for this conservation law, the sum of all independent species (weighted by
+        # the ratio between the coefficient of the species and the species which is elimianted.
         coefs = @view N[i, indepidxs]
-        terms = sum(p -> p[1] / scaleby * p[2], zip(coefs, indepspecs))
```

Comment:
This was probably the function i had to look the most at to understand what was going on. In the end I thought that
  ```
  terms = sum(coef / scaleby * sp for (coef, sp) in zip(coefs, indepspecs))
  ```
  (`p[1]` to `coef` and `p[2]` to `sp`)
  was clearer enough that I figured I could propose the change.